sys/include/net/conn/ip.h

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_conn_ip     Raw IPv4/IPv6 connections
+ * @ingroup     net_conn
+ * @brief       Connection submodule for raw IPv4/IPv6 connections
+ * @{
+ *
+ * @file
+ * @brief   Raw IPv4/IPv6 connection definitions
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef NET_CONN_IP_H_
+#define NET_CONN_IP_H_
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#ifdef MODULE_GNRC_CONN_IP
+#include "net/gnrc/conn.h"
```

Comment:
why this include? The generic conn interface should not care about a GNRC specific header, right?